shaping/output_test.go

```diff
@@ -91,7 +91,7 @@ func TestRecalculate(t *testing.T) {
 		Direction di.Direction
 		Input     []shaping.Glyph
 		Output    shaping.Output
-		Error     error
```

Comment:
Was this line really required too? seems kinda limiting

Reply:
I couldn't see a simple change that didn't require this. What we want to achieve is checking whether the concrete type of a returned error matches the expected concrete type. That's actually kinda a pain. There's likely a way to do it with reflection, but there's no simple mechanism available. I figured that since we only need to check for one concrete type right now, we can do this. If we introduce other concrete types into the API, we can then bite the complexity bullet of implementing the reflect-based type matching.